modules/nodes-cluster-enabling-features-about.adoc

```diff
@@ -23,14 +23,15 @@ The following Technology Preview features are enabled by this feature set:
 ** CSI automatic migration. Enables automatic migration for supported in-tree volume plugins to their equivalent Container Storage Interface (CSI) drivers. Supported for:
 *** Azure File (`CSIMigrationAzureFile`)
 *** VMware vSphere (`CSIMigrationvSphere`)
-** Shared Resources CSI Driver and Build CSI Volumes in OpenShift Builds. Enables the Container Storage Interface (CSI) (`CSIDriverSharedResource`).
-** CSI volumes. Enables CSI volume support for the {product-title} build system (`BuildCSIVolumes`).
-** Swap memory on nodes. Enables swap memory use for {product-title} workloads on a per-node basis (`NodeSwap`).
-** cgroups v2. Enables cgroup v2, the next version of the Linux cgroup API (`CGroupsV2`).
-** crun. Enables the crun container runtime (`Crun`).
-** Insights Operator. Enables the Insights Operator, which gathers {product-title} configuration data and sends it to Red Hat (`InsightsConfigAPI`).
+** Shared Resources CSI Driver and Build CSI Volumes in OpenShift Builds. Enables the Container Storage Interface (CSI). (`CSIDriverSharedResource`)
```

Comment:
There seems to be some funky capitalization going on in this line that might be fix-worthy

Reply:
I looked into this and it looks like "Shared Resources CSI Driver", "Build CSI Volumes", "OpenShift Builds", and "Container Storage Interface" are correct, per how they are referenced elsewhere in the doc set.